<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panser</class>
 <widget class="QWidget" name="Panser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panser</string>
  </property>
  <widget class="QLabel" name="lURL">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="lDireccion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direccion en memoria</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leURL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDireccion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGuardar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teHTML">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>721</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDibujar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
